<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Побудувати контрприклад</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>631</width>
     <height>481</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="rowCount">
    <number>4</number>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
  </widget>
  <widget class="QPushButton" name="btnBuild">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>10</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Перевірити умови верифікації програми</string>
   </property>
   <property name="text">
    <string>Побудувати</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCond">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Показати умову</string>
   </property>
   <property name="text">
    <string>Умова</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOptions">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Показати системні параметри</string>
   </property>
   <property name="text">
    <string>Параметри</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
